bootstrap.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type bootstrapData struct {
+	Clusters  []byte            `json:"clusters"`
+	Listeners []byte            `json:"listeners"`
+	Endpoints map[string][]byte `json:"endpoints"`
+}
+
+func readBootstrapData(rawData []byte) (*bootstrapData, error) {
+	var bsData bootstrapData
+
+	err := json.Unmarshal(rawData, &bsData)
+	if err != nil {
+		return nil, err
+	}
+
+	return &bsData, nil
+}
+
+func (b *bootstrapData) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	switch req.URL.Path {
+	case "/v2/discovery:endpoints":
+		b.handleEDS(w, req)
+	case "/v2/discovery:listeners":
+		b.handleLDS(w, req)
+	case "/v2/discovery:clusters":
+		b.handleCDS(w, req)
+	case "/healthz":
```

Comment:
healthz should change into something that's more actively related to "is envoy ready".

Which is going to mean:

* Obviously, is envoy itself running
* Has envoy fetched all of the xds endpoints at least once